remoteapi/pom.xml

```diff
@@ -145,7 +139,6 @@
               <artifactSet>
                 <includes>
                   <include>com.google.api-client:google-api-client:*</include>
-                  <include>com.google.appengine:proto1:*</include>
```

Comment:
If we remove proto1 from repackaging this fatajar, I am pretty sure we will need to add some classes from the proto2 component, maybe not all but some.  Keep a TODO in your log as a next step